src/components/Admin/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import {
+  Container,
+  FormGroup,
+  Label,
+  Alert,
+  Button,
+  Jumbotron,
+  Row
+} from 'reactstrap';
+import bcrypt from 'bcryptjs';
+import '../../styles/ApproveIndex.css';
+import ApproveComponent from './ApproveComponent';
+
+class Approve extends Component {
```

Comment:
`AdminLogin` as name?

Reply:
changed to AdminHome, login in part of index